app/src/androidTestE2eTest/java/io/appium/uiautomator2/unittest/test/DeviceCommandsTest.java

```diff
@@ -22,6 +22,7 @@
 import org.json.JSONArray;
 import org.json.JSONException;
 import org.json.JSONObject;
+import org.junit.Ignore;
```

Comment:
is it used?

Reply:
The run time of the **scrollVeryLongListSuccessfully** test is about 8.5 minutes, which might fail automated build jobs that have timeout of 10 minutes. To verify this scenario, I suggest ignoring this test in automated builds by adding the @Ignore attribute to it, and only running the test locally.

Reply:
I see that only `shouldShutdownServerOnPowerDisconnect ` test has this attribute. Also, it would be useful to put a comment with the explanation above.

Reply:
Explanation about the long run time has been added to test **scrollVeryLongListSuccessfully**. 
The **shouldShutdownServerOnPowerDisconnect** was added the @Ignore attribute for local runs only, and was checked in by mistake. It should not be in the production code. Removed.